tex/gregoriotex.lua

```diff
@@ -120,6 +120,37 @@ local translation_mark = 1
 local abovelinestext_mark = 2
 log("marker whatsit id is %d", marker_whatsit_id)
 
+local function gregorio_exe()
```

Comment:
Wait, rereading your comment above, that is what you were trying to do, so clearly I've misunderstood something.  Allow me to examine the code a bit further.